core/backend/src/BlobContainerService.ts

```diff
@@ -45,26 +45,21 @@ export namespace BlobContainer {
     iTwinId: Id64String;
     /** optionally, an iModelId within the iTwin. If present, container is deleted when the iModel is deleted. */
     iModelId?: Id64String;
```

Comment:
I’d leave this. We will need containers owned by individuals.

Reply:
Yes, but the owner identity needs to come via OIDC - otherwise it could be spoofed.

Wouldn't it be sufficient to say that the user who created the container is always the initial "owner"?